python/sglang/backend/runtime_endpoint.py

```diff
@@ -125,6 +125,11 @@ def generate(
         else:
             raise RuntimeError(f"Invalid dtype: {sampling_params.dtype}")
 
+        for item in ["return_logprob", "logprob_start_len", "top_logprobs_num", "return_text_in_logprobs"]:
```

Comment:
move this to `sampling_params.to_srt_kwargs()`

Reply:
logprob params should not be included under `data["sampling_params"]` according to the [docs](https://github.com/sgl-project/sglang/blob/main/docs/sampling_params.md)